src/main/java/lotto/utils/StringAddCalculator.java

```diff
@@ -0,0 +1,76 @@
+package lotto.utils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringAddCalculator {
+
+    private final static Pattern pattern = Pattern.compile("//(.)\n(.*)");
+    public final static String DEFAULT_SPLIT_REGEX = ",|:";
+
+    public void stringCalculator(String inputString) {
+        int sum = getNumberList(inputString);
+    }
+
+    public int getNumberList(String inputString) {
+        String args = getCheckInputString(inputString);
+        List<Integer> list = splitInputString(args);
+        return list
+                .stream()
+                .mapToInt(Integer::intValue)
+                .sum();
```

Comment:
```suggestion
        return list.stream()
                .mapToInt(Integer::intValue)
                .sum();
```
한 라인에 점을 하나 쓰도록 한다면 위와 같은 형태가 될 수 있을 것 같아요! :)